torch/csrc/jit/python/init.cpp

```diff
@@ -694,6 +694,29 @@ void initJITBindings(PyObject* module) {
             std::tie(data, size) = self.getRecord(key);
             return py::bytes(reinterpret_cast<const char*>(data.get()), size);
           })
+      .def(
+          "get_storage_from_record",
+          [](PyTorchStreamReader& self,
+             const std::string& key,
+             size_t numel,
+             py::object dd) {
+            at::DataPtr data;
+            size_t size;
+            std::tie(data, size) = self.getRecord(key);
```

Comment:
`auto [data, size] = self.getRecord(key);`

Reply:
Actually, you aren't using size? Maybe just do 

`auto data = self.getRecord(key).get(0); ?`